src/packaging/kpatch/commit_to_overlay.sh

```diff
@@ -445,9 +448,17 @@ for change in ${COMMIT_SHAS[@]}; do
   echo "Generating the kernel overlay"
   cd src/packaging/kpatch
 
-  export GITDATE="${gitdate}" GITAUTHOR="${gitauthor}" COMMIT_FILE="${commit_file}" \
-         LINUX_SRC=${LINUX_SRC}
-  make prepare kernel-overlay >> ${build_log_file}
+  export LINUX_SRC=${LINUX_SRC}
+  make prepare >> ${build_log_file}
+
+  WORK_DIR=work/kvdo-* # Location of the prepared kernel products
+  rm -rf ${LINUX_VDO_SRC}
+  mkdir -p ${LINUX_VDO_SRC}
+  cp -r ${WORK_DIR}/dm-vdo/* ${LINUX_VDO_SRC}
+  sed -i -E -e 's/(#define	CURRENT_VERSION).*/\1 "8.3.0.65"/' \
```

Comment:
This CURRENT_VERSION value is only used to put a version on the VDO kernel module. But once we're in the kernel, having a version number separate from the kernel's version number makes no sense. In the medium-to-long run, we should just remove this version altogether, but it's not a high priority so it might take us a little while to get to. (I've logged a new ticket, VDO-5619, to record that we should do that.)

For this script, the reason to do this is so that we do not constantly generate a diff when applying patches to upstream. (The version is defined as a placeholder, which prepareSourceDist replaces with an actual version string. However, the vdo baseline was generated off of Argon, which has a different version number that vdo-devel, so the generate version number is also not correct.)